packages/xo-web/src/xo-app/tasks/index.js

```diff
@@ -105,6 +105,21 @@ const COLUMNS = [
     name: _('progress'),
     sortCriteria: 'progress',
   },
+  {
+    itemRenderer: task => <FormattedRelative value={task.created * 1000} />,
+    name: _('taskStarted'),
+    sortCriteria: 'created',
+  },
+  {
+    itemRenderer: task => {
+      // [NEED TO TEST]
+      const now = Date.now()
+      const timeNeeded = (now - task.created * 1000) / (task.progress * 100)
```

Comment:
My attempt :stuck_out_tongue: 

```suggestion
      const { progress } = task
      if (progress === 0 || progress === 1) {
        return; // not yet started or already finished
      }
      const elapsed = (now - task.created * 1000)
      const timeNeeded = elapsed / (1 - progress)
```